### PR TITLE
chore: update download as yaml tooltip style

### PR DESCRIPTION
**Pull request by boojack**
After: 
image

### CODE REVIEW

**Comment by tianzhou** [comment]
Defer to @ecmadao

**Review by ecmadao — DISMISSED** [review]
LGTM

File: components/SchemaSystem/SchemaConfigurationPage.vue [hunk 1]
```diff
@@ -16,17 +16,43 @@
...
+                tag="span"
+                class="tooltip-content w-full flex flex-row justify-start items-center"
+              >
+                <template #link>
+                  <a
+                    href="https://github.com/marketplace/actions/sql-review"
+                    class="ml-1 flex flex-row justify-start items-center underline hover:opacity-80"
+                    target="_blank"
+                    @click.stop
+                    >GitHub Action SQL Review<svg
```
Comment (ecmadao) [inline components/SchemaSystem/SchemaConfigurationPage.vue:32]: SQL Review GitHub Action?

Comment (boojack) [reply components/SchemaSystem/SchemaConfigurationPage.vue]: SG, updated. PTAL

**Review by ecmadao — APPROVED** [review]
LGTM

**Pull request merged**